Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocListControlObjectTest.cs

```diff
@@ -651,7 +685,7 @@ public void TestGetRowMatchesExactly ()
               .With.Message.EqualTo(
                   AssertionExceptionUtility.CreateControlMissingException(
                           Driver,
-                          "Unable to find css: .bocListTable .bocListTableBody .bocListDataRow .bocListDataCell[data-boclist-cell-index='7'] span[data-boclist-cell-contents='EO']")
+                          "Unable to find css: .bocListTable .bocListTableBody .bocListDataRow .bocListDataCell[data-boclist-cell-index='7'] *[data-boclist-cell-contents='EO']")
```

Comment:
Service comment: if we do update the selector (see line 238 in the BocListControlObject), this needs updating.